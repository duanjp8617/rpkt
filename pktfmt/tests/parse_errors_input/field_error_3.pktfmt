packet FakeProto {
    header = [
        fake_field1 = Field {
            bit = 8, 
            arg = u16,
        }
    ]
}

