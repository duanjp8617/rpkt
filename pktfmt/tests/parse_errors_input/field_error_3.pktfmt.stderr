ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/field_error_3.pktfmt 3:23-6:9
                          ~~~~~~~
3 |         fake_field1 = Field {
4 |             bit = 8, 
5 |             arg = u16,
6 |         }
    ^^^^^^^^^
field error 3:
invalid arg u16 under repr u8
